geocoding_platform_interface/lib/src/models/placemark.dart

```diff
@@ -34,6 +36,9 @@ class Placemark {
     this.subThoroughfare,
   });
 
+  /// The full address associated with the placemark.
+  final String? address;
+
```

Comment:
Can this be placed in a different PR?

Reply:
Hi @jogboms I looked and there seems to be other PR that already addressed this issue  #60 . Can I know why is it not being review and merged? If there is a an issue I can create a fresh PR for it.

Reply:
Not entirely sure why it hasn't been reviewed and merged yet. I am also just an outside contributor